src/main/java/org/terracotta/statistics/derived/latency/MaximumLatencyHistory.java

```diff
@@ -0,0 +1,125 @@
+/*
```

Comment:
I think a concurrency test could be a good idea as well

Reply:
I thought about that but I do not see what to test the points to pay attention since they are hard to test and really specific use cases. Let me know if yo use how I would add some. There were none also for the previous sample moving average stat so we can had some to it also.

I think what we have to verify is that concurrent threads:

1. cannot create 2 new accumulators A and B where abs(A.start - B.start) < windowSize
2. insertion is done in order in the archive
3. thread de-scheduling could cause a latency to be added to the next created accumulator if 2 threads comes at the end of the window.

I have no clue how to test that...

Reply:
It's possible but I don't have the time to work on it now. So let's keep it for later.